applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/prj_dongle.conf

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# debug: Dongle version
+#
+# This debug build type configuration acts as a dongle.
+#
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+CONFIG_CAF_INIT_LOG_KEEP_ALIVE_EVENTS=n
+
+CONFIG_DESKTOP_ROLE_HID_DONGLE=y
+CONFIG_DESKTOP_DEVICE_PID=0x52DC
+CONFIG_DESKTOP_HID_BOOT_INTERFACE_KEYBOARD=y
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+
+CONFIG_DESKTOP_USB_ENABLE=y
+# Enable the only USB stack that supports USBHS
+CONFIG_DESKTOP_USB_STACK_NEXT=y
+
+# The LLPM must be explicitly enabled, as the Bluetooth Controller
+# is not part of the main application.
+CONFIG_CAF_BLE_USE_LLPM=y
+
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0000
+CONFIG_DESKTOP_BLE_NEW_PEER_SCAN_REQUEST=y
+CONFIG_DESKTOP_BLE_NEW_PEER_SCAN_ON_BOOT=y
+CONFIG_DESKTOP_BLE_PEER_ERASE=y
+
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE=y
+
+################################################################################
+# Zephyr Configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=3584
+CONFIG_ISR_STACK_SIZE=2560
+CONFIG_MAIN_STACK_SIZE=2816
+CONFIG_BT_RX_STACK_SIZE=2048
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=1536
+CONFIG_IDLE_STACK_SIZE=512
+
+CONFIG_BOOT_BANNER=n
+CONFIG_NCS_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_HEAP_MEM_POOL_SIZE=1024
```

Comment:
Is it save to lower this so much?

Reply:
Discussed offline. Increased to `2048` for extra safety (the value of `2048` was introduced for nRF5340 to prevent Event Manager OOM errors).